packages/common/src/ticker.ts

```diff
@@ -59,13 +59,20 @@ export class Ticker {
     let ticker = document.createElement("div");
     ticker.classList.add("en__component");
     ticker.classList.add("en__component--ticker");
-    let str = '<div class="ticker">';
+    let str = `<div class="ticker">`;
     for (let i = 0; i < items.length; i++) {
       str += '<div class="ticker__item">' + items[i] + "</div>";
     }
     str = '<div id="engrid-ticker">' + str + "</div></div>";
     ticker.innerHTML = str;
     this.tickerElement?.parentElement?.insertBefore(ticker, this.tickerElement);
     this.tickerElement?.remove();
+    let tickerSelect = document.querySelector(".ticker");
+    console.log(tickerSelect);
+    let tickerWidth = (tickerSelect) ? getComputedStyle(tickerSelect).width : '1000';
+    tickerWidth = Math.round(parseInt(tickerWidth)).toString();
+    console.log(tickerWidth);
```

Comment:
Please, remove the console.log.
If you want, you can implement the `Logger` component, which will add the debug info to the console only if `debug=true` is present in the URL.
Let me know if you need help doing that!